scripts/tests_extension_hook.py

```diff
@@ -30,5 +30,8 @@ def django_plugin_hook(test_item: YamlTestItem) -> None:
         if "[mypy.plugins.django-stubs]" not in test_item.additional_mypy_config:
             test_item.additional_mypy_config += django_settings_section
 
+    # Disable caching in tests to allow parallel execution
+    test_item.disable_cache = True
```

Comment:
I am not sure that we want to do that. Cache is very important in mypy. It is not a good idea to disable it :(

Reply:
Indeed, it was doing a lot more than I thought, i kinda forgot it also caches a lot of thing from stlib and dependencies. Thanks for pointing that out!